Gemfile

```diff
@@ -19,7 +19,7 @@ gem "minima", "~> 2.0"
 
 # If you have any plugins, put them here!
 group :jekyll_plugins do
-  gem "jekyll-feed", "~> 0.6"
+  gem "jekyll-feed", "~> 0.16"
```

Comment:
This looks like a big revision number jump, from 0.6 to 0.16. So sounds like something @mudge should look at before we merge.

Reply:
While the version constraint here makes it look like we're going from 0.6 to 0.16, the lockfile below shows we're going from 0.15.1 to 0.16 which is a much smaller version bump. As per [semantic versioning](https://semver.org), this _should_ be backwards compatible and safe to merge.